api/resolvers/item.js

```diff
@@ -724,6 +724,18 @@ export default {
         return item
       }
     },
+    updateNoteId: async (parent, { id, noteId }, { models }) => {
+      if (id === undefined) {
+        throw new Error('ID is required for updateNoteId')
```

Comment:
Or because you should indeed change this to:

```suggestion
        throw new GraphQLError('id required', { extensions: { code: 'BAD_INPUT' } })
```

you can also fix the above issue, I think :)

Some unsolicited advice: When contributing to any project, try to find similar pieces of code like you're trying to implement in the code base. And then try to imitate those pieces such that the code does not end up looking like a patchwork family over time, lol :)